src/index.js

```diff
@@ -9,11 +9,9 @@ import './index.css'
 
 const root = ReactDOM.createRoot(document.getElementById('root'))
 root.render(
-  <React.StrictMode>
-    <Provider store={store}>
-      <BrowserRouter>
-        <App />
-      </BrowserRouter>
-    </Provider>
-  </React.StrictMode>
```

Comment:
we should be leaving `React.StrictMode` in. it helps find errors etc, if there's some problem caused by it rendering components twice we should either document and ignore it if it's minor, or fix the problem.